src/tools/check-versions/Makefile

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include ../../../utils.mk
+
+PROJECT_NAME = Kata Containers
+PROJECT_URL = https://github.com/kata-containers
+PROJECT_COMPONENT = check-versions
+
+TARGET = $(PROJECT_COMPONENT)
+
+VERSION_FILE := ./VERSION
+VERSION := $(shell grep -v ^\# $(VERSION_FILE))
```

Comment:
fwics, the main Kata version isn't being used as `cargo run -- --version shows the Cargo.toml version for the tool (`0.1.0`), so I'd either remove these two lines (as the `grep` fails at `cargo build` time), or rework the version handling which would include adding a symlink to the main version file:

```bash
$ ln -s ../../../VERSION
$ git add ./VERSION
```